.chloggen/fix-labels-version-reflects-image-tag.yaml

```
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: labels
```

I think this should be 
```suggestion
component: [collector, target allocator, opamp]
```

But not sure if multiple components are supported...